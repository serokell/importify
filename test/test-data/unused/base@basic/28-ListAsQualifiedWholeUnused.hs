module ListAsQualifiedWholeUnused where

import qualified Data.List as L

main :: IO ()
main = pure ()
