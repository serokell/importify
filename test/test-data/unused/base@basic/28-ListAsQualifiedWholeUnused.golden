module ListAsQualifiedWholeUnused where


main :: IO ()
main = pure ()
